tests/test_serializer.py

```diff
@@ -68,8 +67,25 @@ def test_serialize_sets(extra_normalizer):
 
 def test_serialize_custom_mapping(extra_normalizer):
     class CustomReprDict(dict):
-        def sentry_repr(self):
+        def __sentry_repr__(self):
             return "custom!"
 
     result = extra_normalizer(CustomReprDict(one=1, two=2))
     assert result == "custom!"
+
+
+try:
+    from unittest import mock
+except ImportError:
+    pass
+else:
```

Comment:
I would use pytest.importorskip here, or generally use pytest-skipping instead of letting the test be undefined.